### ISSUE TITLE
Attention Visiualization?

### ISSUE THREAD

**Issue by alecda573** [Vegetebird/StridedTransformer-Pose3D#5]
Can you please share the  code you used to visualize your attention maps  for different heads?

**Comment by Vegetebird** [comment]
Hi~I have shared the code to visualize attention maps in "common/plot_att.py".

**Comment by alecda573** [comment]
@Vegetebird thanks for sharing, I do not see where you make predictions and then plot the softmaxes from the attention block… am I missing something?

**Comment by Vegetebird** [comment]
You should save the attention scores after softmax, just like this 
`
    p_attn = F.softmax(scores, dim=-1)
    ## save
    p_attn_save = p_attn.cpu().detach().numpy()
    np.savez_compressed('attn.npz', p_attn_save)
`
